src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelService.java

```diff
@@ -1,45 +1,63 @@
 package com.sprint.mission.discodeit.service.jcf;
 
 import com.sprint.mission.discodeit.collection.Channels;
+import com.sprint.mission.discodeit.dto.channel.ChannelResponse;
+import com.sprint.mission.discodeit.dto.channel.CreateChannelRequest;
 import com.sprint.mission.discodeit.entity.Channel;
 import com.sprint.mission.discodeit.service.ChannelService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
 
-import java.util.Map;
+import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
+import java.util.stream.Collectors;
 
+@Service
+@RequiredArgsConstructor
 public class JCFChannelService implements ChannelService {
     private final Channels channels = new Channels();
```

Comment:
`@RequiredArgsConstructor` 애노테이션은 final 키워드가 붙은 필드 중 초기화되지 않은 필드만을 대상으로 생성자 매개변수로 사용합니다.  

> `@RequiredArgsConstructor` generates a constructor with 1 parameter for each field that requires special handling. **All non-initialized final fields** get a parameter, as well as any fields that are marked as `@NonNull` that aren't initialized where they are declared.  

`@RequiredArgsConstructor`를 사용한다면 Channels 초기화 코드를 제거해주는게 좋을 것 같아요.  

```suggestion
@RequiredArgsConstructor
public class JCFChannelService implements ChannelService {
    private final Channels channels;
```

참고 문서 : https://projectlombok.org/features/constructor